src/components/Forecast.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import ForecastDay from './ForecastDay';
+
+export default class Forecast extends Component {
+  constructor() {
+    super();
+    this.state = {
+      daysToRender: 5,
```

Comment:
can you move this 5 to a constant called `DAYS_TO_RENDER` or some other identifier outside the class?

Reply:
I don't know if this is appropriate, but I created two separate constants for each time frame to render (5 and 3 days respectively) to retrofit it into my click handler